CTIDH/isogeny.py

```diff
@@ -2,12 +2,19 @@
 from math import floor, sqrt
 
 from CTIDH.mont import MontgomeryCurve
-from CTIDH.utils import read_velusqrt_steps_info, hamming_weight, bitlength, isequal, batchmaxprime_of_Li, batchminprime_of_Li, batchnumber_of_Li, CMOV, CSWAP
+from CTIDH.utils import read_prime_info ,read_velusqrt_steps_info, hamming_weight, bitlength, isequal, batchmaxprime_of_Li, batchminprime_of_Li, batchnumber_of_Li, CMOV, CSWAP
 from CTIDH.polymul import PolyMul
 from CTIDH.polyredc import PolyRedc
-
+from CTIDH.primefield import PrimeField
 import numpy
 
+p1024_info = read_prime_info("p1024_CTIDH")
+p2048_info = read_prime_info("p2048_CTIDH")
```

Comment:
Things like read_prime_info and Fp1024 should not appear in this file. If you want to use Fp constant, use self.field(a) instead.